lib/model/rwfolder_test.go

```diff
@@ -329,17 +330,17 @@ func TestDeregisterOnFailInCopy(t *testing.T) {
 	m.AddFolder(defaultFolderConfig)
 
 	emitter := NewProgressEmitter(defaultConfig)
+	m.progressEmitter = emitter
 	go emitter.Serve()
 
 	p := rwFolder{
-		folder:          "default",
-		dir:             "testdata",
-		model:           m,
-		queue:           newJobQueue(),
-		progressEmitter: emitter,
-		errors:          make(map[string]string),
-		errorsMut:       sync.NewMutex(),
+		dir:       "testdata",
+		queue:     newJobQueue(),
+		errors:    make(map[string]string),
+		errorsMut: sync.NewMutex(),
 	}
+	p.folderID = "default"
```

Comment:
Also this is just weird. Why not initialize these attributes when constructing the rwFolder like the others, above.